lib/OpenQA/WebAPI/Controller/API/V1/Bug.pm

```diff
@@ -46,7 +46,8 @@ OpenQA API implementation for bug handling methods.
 Returns a dictionary of bugs reported in the system of the form { id: bug } where the key
 is the ID in the database and the value is the external bug, eg. bsc#123 or poo#123.
 
-The optional parameter "refreshable" limits the results to bugs updated recently.
+The optional parameter "refreshable" limits the results to bugs not updated recently
+that are not known to not exist.
```

Comment:
not updated, not known, not exist - Too many negatives for my taste :-)

Reply:
But this is what the argument does.
It does not return bugs that are known to exist - it returns bugs that are not known to not exist.

Reply:
"… limits the results to potentially stale or unreferenced bug entries"?

Reply:
> "… limits the results to potentially stale or unreferenced bug entries"?

It doesn't care if bugs are unreferenced.
And _stale_ is too generic to be of any value to someone who wants to understand what this function does IMHO.

Reply:
The the sentence with all the "not"s isn't useful to understand what the function does, too. At least I'd just read the code for clarification. In that sense you could remove the sentence altogether.

Reply:
this point is still open

Reply:
In that case I will just leave this as it is.